libs/coin-modules/coin-polkadot/src/test/fixtures.ts

```diff
@@ -0,0 +1,75 @@
+import { CryptoCurrency } from "@ledgerhq/types-cryptoassets";
+import { PolkadotAccount } from "../types";
+import { decodeAccountId } from "@ledgerhq/coin-framework/account";
+import { getDerivationScheme, runDerivationScheme } from "@ledgerhq/coin-framework/derivation";
+import { SubAccount } from "@ledgerhq/types-live";
+import BigNumber from "bignumber.js";
+
+export const makeAccount = (
```

Comment:
there is already a createFixtureAccount, maybe we can do something with it : ledger-live/libs/coin-modules/coin-polkadot/src/types/bridge.fixture.ts

Reply:
Agreed, the only issue is that there's some `faker` in the bridge fixture
We need something more deterministic for the coin tester